### PR TITLE
GPU: [Linux] Prefer dedicated card over integrated card.

### LINKED ISSUE #465: GPU not identified anymore
## Description
Upon installing for the first time (some time in the 1st quarter of this year), my GPU showed in the output of the neofetch command. Now with 1.9.1, my GPU defaults to a generic name `Intel Integrated Graphics` (in the screenshot below).

I have done my own tests to verify my GPU using the command in the script. The output of `$(PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA|/ {print $3 " " $4}')` on my system (formatted) is: 

```
Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor DRAM Controller
Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor PCI Express x16 Controller
Intel Corporation 4th Gen Core Processor Integrated Graphics Controller
Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor HD Audio Controller
Intel Corporation 8 Series/C220 Series Chipset Family USB xHCI
Intel Corporation 8 Series/C220 Series Chipset Family MEI Controller #1
Intel Corporation 8 Series/C220 Series Chipset Family USB EHCI #2 
Intel Corporation 8 Series/C220 Series Chipset High Definition Audio Controller 
Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #1 
Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #2 
Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #3 
Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #4 
Intel Corporation 8 Series/C220 Series Chipset Family USB EHCI #1 
Intel Corporation HM86 Express LPC Controller 
Intel Corporation 8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode] 
Intel Corporation 8 Series/C220 Series Chipset Family SMBus Controller
NVIDIA Corporation GM107M [GeForce GTX 860M]
Intel Corporation Wireless 7260 
Realtek Semiconductor Co., Ltd. RTL8411B PCI Express Card 
Reader Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
```
Line 16 (the 4th to last line) confirms the presence of my NVIDIA GPU.

I then went to the github repo and downloaded tag versions 1.6 (around the time neofetch used to work) through to 1.9.1 and I realized that from 1.6 to 1.8.1, my GPU is shown and refused to show in versions 1.9 and 1.9.1 (screenshots below). I even downloaded the `gpu_fix` tag to no avail.

### Isolated Problem First Occurrence 
I managed to isolate when the problem first occurred by forking the repository and doing a by-tag compare and opening all commits with gpu in their description. I then proceeded to download and run each which revealed the problem lied in the commit [Refactor GPU](https://github.com/zoutiyx/neofetch/commit/cc9095ccd5bcd31383fb51d754cd998d118e16d6). Due to my limited bash knowledge and absence of time, I won't be able to fix it myself. But I'm hoping it can be resolved.

#### Neofetch version
1.9.1

## Screenshots (labeled versions)
**1.9.1**
[](https://postimg.org/image/rqku308uh/)

**1.9**
[](https://postimg.org/image/bgpokv5o5/)

**1.8.1**
[](https://postimg.org/image/aw7n5re49/)

**1.6**
[](https://postimg.org/image/5ox9dp0e3/)

**commit 5dddaa0b260d115d1db2f06c5e6c7838f6e6a48d**
[](https://postimg.org/image/x8lc72szf/)

## Verbose log
if still necessary, posted on [pastebin](http://pastebin.com/n7y5N94F)

### PR DESCRIPTION
**Pull request by dylanaraps**
Title. 

Fixes #465

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -927,29 +927,20 @@ get_gpu() {
...
-
-                if [[ -z "$bdf_number" ]]; then
-                    # Fallback if no kernel driver is in use
-                    gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA/ {print $3 " " $4}')"
-
-                else
-                    # Find the currently used GPU by its BDF
-                    gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -v bdf_number="$bdf_number" -F '\\"|\\" \\"' '$0 ~ bdf_number {print $3 " " $4}')"
-                fi
+                gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA/ {print $3 " " $4}')"
```
Comment (konimex) [inline neofetch]: In some devices (including mine), `3D` is replaced by `Display`. Changing `/3D|VGA/` to `/Display|3D|VGA/` may be beneficial here.

Comment (dylanaraps) [inline neofetch]: Will do, thanks!

**Comment by zoutiyx** [comment]
@dylanaraps,

I apologize for the late reply, but yes, it did fix my issue.

[](https://postimg.org/image/airo78fen/)

**Comment by dylanaraps** [comment]
No worries! :)

**Pull request merged**